wled00/wled20_ir.ino

```diff
@@ -48,15 +48,23 @@ void decodeIR(uint32_t code)
   if (code == 0xFFFFFFFF) //repeated code, continue brightness up/down
   {
     irTimesRepeated++;
-    if (lastValidCode == IR24_BRIGHTER)
+    if (lastValidCode == IR24_BRIGHTER | lastValidCode == IR40_BPLUS )
```

Comment:
long press on different remotes - IR24_ is the existing one and IR40_ is the new one.
No need to include the other remotes here as the IR-code for this button is the same on all new remotes.